static/src/javascripts/bootstraps/enhanced/common.js

```diff
@@ -420,7 +420,7 @@ define([
                 ['c-save-for-later', modules.saveForLater],
                 ['c-show-membership-messages', modules.showMembershipMessages],
                 ['c-email', modules.initEmail],
-                ['c-user-features', userFeatures.refresh],
+                ['c-user-features', userFeatures.refresh, userFeatures],
```

Comment:
Actually, come to think of it...

Could we not just replace this whole PR with -

```javascript
['c-user-features', userFeatures.refresh.bind(userFeatures)],
```

...?